.gitignore

```diff
@@ -39,7 +39,9 @@ gradle/wrapper/gradle/
 gradle/wrapper/gradlew*
 
 # Go native dependencies
-ext/golang/dist
+ext/golang
+go-latest.tar.gz
+go-1.4.tar.gz
```

Comment:
If the real archive names are used (see comments below), this two lines need to be changed to something like `/go*.src.tar.gz`.